api/k8s/v1/k8s.proto

```diff
@@ -509,6 +517,35 @@ message DeleteCronJobRequest {
 message DeleteCronJobResponse {
 }
 
+message ConfigMap {
+  option (clutch.api.v1.id).patterns = {
+    type_url : "clutch.k8s.v1.ConfigMap",
+    pattern : "{cluster}/{namespace}/{name}"
+  };
+
+  string cluster = 1;
+  string namespace = 2;
+  string name = 3;
+
+  map<string, string> data = 4;
+  map<string, string> annotations = 5;
+  map<string, string> labels = 6;
+}
+
+message ListConfigMapsRequest {
+  string clientset = 1 [ (validate.rules).string = {min_bytes : 1} ];
+  string cluster = 2 [ (validate.rules).string = {min_bytes : 1} ];
+  string namespace = 3 [ (validate.rules).string = {min_bytes : 1} ];
+
+  ListOptions options = 4 [ (validate.rules).message = {required : true} ];
```

Comment:
for reference:
https://github.com/lyft/clutch/blob/main/api/k8s/v1/k8s.proto#L189

to be able to use this helper method https://github.com/lyft/clutch/blob/4d0b7faab3f2b5cc4c5b6bb9de84eb52b174d393/backend/service/k8s/meta.go#L12